### PR TITLE
Enhanced rigid contact API

### PR DESCRIPTION
**Pull request by Ipuch**
Enhanced features for rigid contacts iPython API such as :
- position in global frame
- velocity in global frame
- acceleration in global frame

### CODE REVIEW

**Comment by Ipuch** [comment]
I succeeded in compiling in my debug and eigen but not in casadi release.

File: src/RigidBody/Contacts.cpp [hunk 1]
```diff
@@ -45,6 +49,23 @@ unsigned int rigidbody::Contacts::AddConstraint(
...
+        axis += "x";
+    }
+    if (has_y != 0){
+        axis += "y";
+    }
+    if (has_z != 0){
+        axis += "z";
+    }
+
+    m_rigidContacts->push_back(NodeSegment(body_point, name, "",true,false, axis.c_str(),body_id));
```
Comment (pariterre) [inline src/RigidBody/Contacts.cpp:68]: This is the best I could do... but I am not sure this is okay, could you add a test to validate that when you call this function it actually adds the axis properly?

**Review by pariterre — CHANGES_REQUESTED** [review]
GG, just a very quick test and we are good to go

File: src/RigidBody/Contacts.cpp (hunk 1 above)
Comment (pariterre) [reply src/RigidBody/Contacts.cpp:68]: this works

**Review by pariterre — APPROVED** [review]
LGTM

**Pull request merged**